ANS_JS_template.tex

```diff
@@ -188,116 +188,64 @@ \subsection{Other Research on NRHES Models}
+The underlying structure of passing materials between various facilities is similar enough between nuclear materials in a fuel cycle and heat or electricity flow in a NRHES that a NFCS could model a nuclear renewable hybrid energy system. In the case of a NRHES model, the facilities are the NPP, the renewable, and the industrial process at least.  In the case of the nuclear fuel cycle, the facilities are the steps in the given fuel cycle. Noting the capabilities found desirable by end users in the EPRI 2016 report, all of the criteria besides from the ability to track many nuclides could also be applied to a NRHES. Both systems are time dependent and include uncertainty measures. Many of the uncertainty measures found in NFCSs are the same for a NRHES including O\&M costs, water costs, and challenges in siting a nuclear power plant. Optimally, both systems would be:
 \begin{itemize}
 
-\item flexible so that new component models and specific regional characteristics could easily be added, 
-\item open source so that researchers can use the tools instead of rebuilding already available tools, and 
-\item usable by technical and non technical people alike to address policy decisions. 
+\item flexible so that new component models and specific regional characteristics could easily be added
+\item well documented and accessible
+\item include uncertainty/sensitivity analysis capabilities
+\item usable by technical and non technical people alike to address policy decisions.
 
 \end{itemize}
- 
-A NFCS would need to be able to reasonably model the dynamic nature of NRHES. The dynamics aspects of concern, at this point in the research, are how the energy would switch between different loads as well as the ramping of various generation components.  The nuclear fuel cycle does not require the quantities to be exchanged nearly as regularly as a nuclear hybrid energy system and with as inconsistent of quantities. The dynamic impacts on the physical system would need to be included in order to generate a meaningful model of a NRHES. The stochastic aspect of the renewables components of a NRHES would also require either external coupling with a tool able to model randomness such as RAVEN or the use of real time data from a specific region. The dynamic and stochastic components of modeling a nuclear hybrid energy system would take some work to incorporate. 
 
```

Comment:
Deleted this section as I am no longer making the argument that it makes sense to model a NRHES with a NFCS.